src/main/java/codegym/tequila/fisioapp/model/Therapist.java

```diff
@@ -0,0 +1,86 @@
+package codegym.tequila.fisioapp.model;
+
+import jakarta.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "therapists")
+public class Therapist {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
We are not using auto-generated ids, please use UUID and set it from the service